2-WebApp-graphapi-on-behalf/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Was committing `.vscode` cache folder intentional? We might want to add this to `.gitignore` if not.

Reply:
Not intentional.  How would we specify the .vscode folder or the settings.json file in .gitignore?  Sorry, I'm not that familiar.

Reply:
Sure thing. Just adding the file names to the `.gitignore` file should do it. More on this [here](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files). 

You'll need to delete the .vscode folder and commit that first though, so that its removed from git history